tools/xcarchivetool/make_xcarchive.py

```diff
@@ -135,6 +145,14 @@ def _main(
       os.path.join(dsyms_path, os.path.basename(dsym)),
       dirs_exist_ok=True)
 
+  # Copy the linkmap files into the .xcarchive.
+  linkmaps_path = os.path.join(output_path, "Linkmaps")
+  os.makedirs(linkmaps_path, exist_ok=True)
```

Comment:
Can you avoid this by following the dSYM `dirs_exist_ok` param usage in the copy call?

Reply:
Will do

Reply:
Linkmaps are single files though, copyfile does not take that parameter

Reply:
Linkmaps are single files though, that argument does not work for copyfile

Reply:
🤦 Makes sense. Can you avoid creating the directory unless linkmap files are present?